src/Tracker/trackerData.tsx

```diff
@@ -503,4 +464,432 @@ export const analyticsEventsMapping = new Map<
     'screen_welcome',
     { eventName: 'WELCOME', properties: { event_type: 'screen' } },
   ],
+  [
+    'screen_document_front_confirmation_cutoff_detected',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { cutoff: 'warning' },
```

Comment:
What exactly do we mean by 'warning'? This alert is not necessarily going to be a warning from UI POV. The user can only skip these alerts after two attempts so from their prospective this is an error. Just want to understand why we are sending "warning" or "error" for the UI alerts.

Reply:
I think the initial reasoning was because it doesn't get the user stuck, as it only tries it twice, but I agree that it's received as an error by the user.

Thinking about it, I would think about a warning as a non interruptive message that they can ignore or dismiss and an error as something (temporary) blocking or interrupting their flow. What do you think?

There are a couple of "errors" defined as warnings or are dismissible, like camera inactivity (can be dismissible) and recording time out (with the action reset). I would label them as warnings for example.

Reply:
Yes, I agree. I would consider an error anything that is preventing the user from proceeding and cannot be dismissed